package.json

```diff
@@ -87,15 +87,17 @@
     "styled-components": "^5.3.6",
     "ts-jest": "^29.0.5",
     "ts-node": "^10.9.1",
-    "typescript": "^4.5.2"
+    "typescript": "^4.5.2",
+    "react-native-gesture-handler": "^2.9.0"
```

Comment:
the version should be 2.13.3

Reply:
Also, we need to make sure the version in `rn-base-component` and `rn-base-project/rn-base-project-typescript` is the same

Reply:
Hi @baonguyenhsts  Hmm I understand but we already included `^` which means as long as we yarn install, it will get latest from version 2.x.x so actually we don't need to change it everytime it have new version with 2.x.x

Reply:
Okay, I just need to ensure that their versions are identical.